core/src/main/java/org/apache/accumulo/core/clientImpl/ThriftTransportPool.java

```diff
@@ -44,8 +50,11 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.Preconditions;
-import com.google.common.collect.Iterables;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+@SuppressFBWarnings(value = "UL_UNRELEASED_LOCK",
```

Comment:
Do you know if moving this suppression to the shutdown() method would work?  If so, making it more narrow would be nice.

Reply:
Yes it does apparently. Just moved it.